tests/test_wallet.py

```diff
@@ -113,29 +154,55 @@ async def test_transfer_nft(self):
         txs = await self.wallet.transfer_nft(
             "0x995a8b7f96cb837533b79775b6209696d51f435c",
             first_value,
-            "USDC"   
+            "USDC"
         )
-
-        assert txs
+        self.assertEqual(len(txs), 2)
+        for i, tr in enumerate(txs):
+            try:
+                status = await tr.await_committed(attempts=100)
+                self.assertEqual(status, TransactionStatus.COMMITTED)
+            except Exception as ex:
+                assert False, f"test_transfer_nft, transaction {i} has failed with error: {ex}"
 
     async def test_withdraw_nft(self):
-        await self.wallet.mint_nft("0x0000000000000000000000000000000000000000000000000000000000000123",
-                                   "0x21dDF51966f2A66D03998B0956fe59da1b3a179F", "USDC")
+        """
+        # Looks like there was misunderstandings, using external account but getting mint_nft from this wallet
```

Comment:
It is only possible to withdraw an nft only after it was verified (so one of the previous tests should be about minting an nft and waiting for its verifyReceipt)

Reply:
Ok. I did the testing and could not get the verified state from server( 4-5 minutes - 3000 attempts with 1 second delay). 
In overall. Your suggestions : should I add the different test in this case?

Reply:
Oh, I see. Verification does take a lot of time. I think that for now the following would work:

- Check if there is an nft to withdraw. 
- If there is none, skip. 

But please leave the comment why this choice was made

Reply:
I refactored 2 test cases and put the idea into the doc string